tests/test_hazard.py

```diff
@@ -13,30 +15,58 @@
     # },
 
     "fiat_objects": {
-        "folder"   : "test_hazard",
+        "folder"   : "test_hazard_1",
         "ini"      : "test_hazard.ini",
         "catalog"  : "fiat_catalog_hazard.yml",
 
-    },
+    }
 }
 
 @pytest.mark.parametrize("case", list(_cases.keys()))
-
 def test_Hazard(case):
     # Read model in examples folder.
     root             = DATASET.joinpath(_cases[case]["folder"])
     config_fn        = DATASET.joinpath(_cases[case]["ini"])
     data_libs        = DATASET.joinpath(_cases[case]["catalog"])
 
-    hyfm = FiatModel(root=root, mode="r", data_libs=data_libs, config_fn=config_fn)
+    hyfm = FiatModel(root=root, mode="w", data_libs=data_libs, config_fn=config_fn)
 
-    raster_max_depth    = hyfm.data_catalog.get_rasterdataset("max_depth")
-    flood_maps_varaible = hyfm.data_catalog.get_rasterdataset("flood_maps_varaible")
-    hazard_type         = hyfm.get_config('setup_config', 'hazard_type')
+    map_fn         = configread(config_fn)['setup_hazard']['map_fn']
+    map_type       = configread(config_fn)['setup_hazard']['map_type']
+    rp             = configread(config_fn)['setup_hazard']['rp']
+    crs            = configread(config_fn)['setup_hazard']['crs']
+    nodata         = configread(config_fn)['setup_hazard']['nodata']
+    var            = configread(config_fn)['setup_hazard']['var']
+    chunks         = configread(config_fn)['setup_hazard']['chunks']
+    maps_id        = configread(config_fn)['setup_hazard']['maps_id']
+    name_catalog   = configread(config_fn)['setup_hazard']['name_catalog']
+    risk_output    = configread(config_fn)['setup_hazard']['risk_output']
+    hazard_type    = configread(config_fn)['setup_config']['hazard_type']
 
-    hyfm.setup_hazard()
+    hyfm_hazard = Hazard()
 
-    assert hyfm
+    hyfm_hazard.checkInputs(
+        hyfm,
+        map_fn,
+        map_type,
+        chunks,
+        rp,
+        crs,
+        nodata,
+        var,
+    )
 
+    ds = hyfm_hazard.readMaps(
+        hyfm,
+        name_catalog,
+        hazard_type,
+        risk_output,
+        crs,
+        nodata,
+        var,
+        chunks,
+    )
+    ds.to_netcdf("C:/Users/fuentesm/CISNE/HydroMT_sprint_sessions/test_77.nc")
```

Comment:
Could you remove the local path in the test and save the file with the function
`hyfm.write()`